Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.8-alpine3.6
+
+ENV CGO_ENABLED=0
+
+RUN apk add --no-cache bash make zip chromium nodejs-npm && \
+    npm install -g browserify && \
```

Comment:
this better just use `yarn` instead of `npm install ...`, it will fetch all needed dependencies of the specific fixed versions.

Reply:
Im not into this js things very much...just installing and running `yarn` does not seem to work as I'm still missing `browserify` afterwards.

Reply:
does it produce any output? the difference is that `npm install` will install the very latest version, while `yarn` will install the specific version that I tested as working.

try `yarn` or `yarn install`, but I think it is the same command.

Reply:
I see, `yarn` installs binary's to `./node_modules/.bin` rather that a "global" path.